singleFrameProcCoreTasks/charimg.rst

```diff
@@ -0,0 +1,304 @@
+#####################
+CharacterizeImageTask
+#####################
+
+Given an exposure with defects repaired (masked and interpolated over,
+e.g. as output by :doc:`IsrTask <isrtask>`), this task does initial
+source extraction and PSF estimation.
+
+
+Its primary functions are to:
+
+  - Detect and measure bright sources
+
+  - Repair cosmic rays
+
+  - Measure and subtract background
+
+  - Measure the PSF
```

Comment:
Again, it's about precision.  We need to be brief but correct.  I think it's fundamental to the algorithm that it is iterative.  I.e. you can't measure without a PSF, but you need detections to measure the PSF.